src/tracks/playabletrack/notetrack/ui/NoteTrackView.cpp

```diff
@@ -723,7 +736,20 @@ void NoteTrackView::Draw(
       const auto hasSolo = artist->hasSolo;
       muted = (hasSolo || nt->GetMute()) && !nt->GetSolo();
 #endif
+
+#ifdef EXPERIMENTAL_NOTETRACK_OVERLAY
+      TrackArt::DrawBackgroundWithSelection(context, rect, nt.get(), AColor::labelSelectedBrush, AColor::labelUnselectedBrush);
+      
+      bool selected{ false };
+      if (auto affordance = std::dynamic_pointer_cast<NoteTrackAffordanceControls>(GetAffordanceControls()))
+      {
+         selected = affordance->IsSelected();
+      }
```

Comment:
I don't think the lines above should be compiled only when the EXPERIMENTAL is on

Reply:
That is, I mean compile them irrespective of that EXPERIMENTAL